src/Stratis.Bitcoin/Consensus/ChainedHeaderTree.cs

```diff
@@ -591,6 +599,8 @@ public bool BlockDataDownloaded(ChainedHeader chainedHeader, Block block)
             bool partialValidationRequired = chainedHeader.Previous.BlockValidationState == ValidationState.PartiallyValidated
                                           || chainedHeader.Previous.BlockValidationState == ValidationState.FullyValidated;
 
+            this.logger.LogTrace("[BLOCK_DOWNLOAD_PREVIOUS_STATE]{0}:{1}:{2}", nameof(chainedHeader.Previous), chainedHeader.Previous, chainedHeader.Previous.BlockValidationState);
```

Comment:
no need to trace this `chainedHeader.Previous.BlockValidationState` its traced as part of the `Previous` ChainedHeader